Modules/System Tests/Rest Client/app.json

```diff
@@ -0,0 +1,54 @@
+{
+  "id": "ae153cbb-ad55-447c-9226-5af3ef57280f",
+  "name": "AL Rest Client Tests",
```

Comment:
You might want to call this "Rest Client Tests", since the System module is called "Rest Client"

Reply:
The PR had some good improvements added and it's a nice module in general, I gave it another quick round of review and didn't see anything outstanding, so now I'm just looking forward to get a standard module for Http.
Well done and thanks for the contribution!

Reply:
Renamed the test app.